tap_stripe/__init__.py

```diff
@@ -705,6 +705,27 @@ def sync_sub_stream(sub_stream_name, parent_obj, updates=False):
             obj_ad_dict = sub_stream_obj.to_dict_recursive()
 
             if sub_stream_name == "invoice_line_items":
+                # we will get "unique_id" for default API versions older than "2019-12-03"
```

Comment:
Add a general comment here regarding which field's value moves to another field and all or write one sample example for both records(old and new) with changed field values.

Reply:
Added comment.